packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -288,11 +253,24 @@ const ModalScrollContent = styled(Flex)<ModalScrollContentProps>`
   `};
 `
 
-const CloseIconWrapper = styled(Box)`
+// TODO: Consider extracting the equivalent of `TouchableWithoutFeedback`
```

Comment:
So yeah, something I find very useful is basically implementing a `Box` component with a button underlying it. Something like https://github.com/auspices/eos/blob/master/src/Clickable/Clickable.ts